knimin/lib/format.py

```diff
@@ -28,6 +28,49 @@ def format_epmotion_file(volumes, destination):
     return "\n".join(contents)
 
 
+def format_normalization_echo_pick_list(vol_sample, vol_water):
+    """Formats the contents of an echo normalization pick list file
+
+    Parameters
+    ----------
+    vol_sample : 2d numpy array of floats
+        The per well sample volume
+    vol_water : 2d numpy array of floats
+        The per well water volume
+
+    Returns
+    -------
+    str
+        The contents of the echo pick list normalization file
+    """
+    contents = ['Source Plate Name,Source Plate Type,Source Well,'
+                'Concentration,Transfer Volume,Destination Plate Name,'
+                'Destination Well']
+    # write the water transfer values
+    rows, cols = vol_water.shape
+    for i in range(rows):
+        for j in range(cols):
+            well_name = "%s%d" % (chr(ord('A') + i), j+1)
```

Comment:
Yeah, that was on the JavaScript side. I would not worry here at this moment because we don't have a way of representing bigger plates.